libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -714,8 +714,8 @@ void RsGxsNetService::syncWithPeers()
 
             // The last post will be set to TS 0 if the req delay is 0, which means "Indefinitly"
 
-            if(req_delay > 0)
-				msg->createdSinceTS = std::max(0,(int)time(NULL) - req_delay);
```

Comment:
(int)something  *is* a static cast. So why convert all the casts in the source into explicit static<cast>(int)? This makes the whole code much less readable.

Reply:
/libretroshare/src/gxs/rsgxsnetservice.cc:718: warning: use of old-style
cast
/libretroshare/src/gxs/rsgxsnetservice.cc:718: warning: implicit
conversion changes signedness: 'const int' to 'uint32_t' (aka 'unsigned
int')